apps/prairielearn/src/pages/enroll/enroll.html.ts

```diff
@@ -305,3 +182,47 @@ export function EnrollmentLimitExceededMessage({ resLocals }: { resLocals: Recor
     </html>
   `.toString();
 }
+
+function AddCourseModal({ csrfToken }: { csrfToken: string }) {
+  return Modal({
+    id: 'add-course-modal',
+    title: 'Confirm add course',
+    body: html`
+      <p>Are you sure you want to add this course content to your PrairieLearn account?</p>
+      <p>
+        Adding or removing courses here only affects what is visible to you on PrairieLearn. This
+        does not change your university course registration.
+      </p>
+    `,
+    footer: html`
+      <input type="hidden" name="__csrf_token" value="${csrfToken}" />
+      <input type="hidden" name="course_instance_id" class="js-course-instance-id" />
+      <button type="button" class="btn btn-secondary" data-dismiss="modal">Cancel</button>
+      <button type="submit" class="btn btn-danger" name="__action" value="enroll">
```

Comment:
Why is this `btn-danger`? The original was `btn-info`.

Reply:
Bad copy/paste, sorry. Changed in a1d39e1.